src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationForm.test.tsx

```diff
@@ -393,4 +400,207 @@ describe('PublicationForm', () => {
+      ],
+    };
+
+    const testRelease1: MyRelease = {
+      amendment: false,
+      approvalStatus: 'Approved',
+      contact: {
+        contactName: 'Test contact name',
+        contactTelNo: 'Test contact tel',
+        id: 'test-contact-id',
+        teamEmail: 'test-contact@hiveit.co.uk',
+        teamName: 'Test team name',
+      },
+      id: 'release-id-1',
+      latestInternalReleaseNote: 'Test internal release note',
+      latestRelease: true,
+      live: true,
+      permissions: {
+        canAddPrereleaseUsers: false,
+        canUpdateRelease: false,
+        canDeleteRelease: false,
+        canMakeAmendmentOfRelease: false,
+      },
+      preReleaseAccessList: '',
+      previousVersionId: 'prev-version-id-1',
+      publicationId: 'publication-id-1',
+      publicationSlug: 'publication-slug-1',
+      publicationTitle: 'Publication 1',
+      published: '2021-01-01T11:21:17',
+      releaseName: 'Test release name',
+      slug: 'release-slug-1',
+      timePeriodCoverage: {
+        label: 'Academic year',
+        value: 'AY',
+      },
+      title: 'Release 1',
+      type: 'AdHocStatistics',
+    };
+
+    const testPublication1: MyPublication = {
+      id: 'publication-id-1',
+      title: 'Publication 1',
+      contact: {
+        id: 'contact-1',
+        contactName: 'John Smith',
+        contactTelNo: '0777777777',
+        teamEmail: 'john.smith@test.com',
+        teamName: 'Team Smith',
+      },
+      releases: [testRelease1],
+      legacyReleases: [],
+      methodologies: [],
+      themeId: 'theme-1',
+      topicId: 'topic-1',
+      permissions: {
+        canAdoptMethodologies: true,
+        canCreateReleases: true,
+        canUpdatePublication: true,
+        canUpdatePublicationTitle: true,
+        canUpdatePublicationSupersededBy: true,
+        canCreateMethodologies: true,
+        canManageExternalMethodology: true,
+      },
+    };
+
+    const testPublication2: MyPublication = {
+      ...testPublication1,
+      id: 'publication-id-2',
+      title: 'Publication 2',
+      releases: [],
+    };
+
+    test('does not render the `Archive publication` field when showSupersededBy is false', async () => {
+      themeService.getThemes.mockResolvedValue(testThemes);
+      publicationService.getPublicationSummaries.mockResolvedValue([
+        {
+          id: testPublication1.id,
+          title: testPublication1.title,
+          slug: 'slug-1',
+        },
+      ]);
+
+      render(
+        <PublicationForm
+          initialValues={{
+            title: 'Test title',
+            topicId: 'topic-4',
+            teamName: 'Test team',
+            teamEmail: 'team@test.com',
+            contactTelNo: '0123456789',
+            contactName: 'John Smith',
+          }}
+          showTitleInput
+          showSupersededBy={false}
+          onSubmit={noop}
+        />,
+      );
+
+      await waitFor(() => {
+        expect(screen.getByLabelText('Publication title'));
```

Comment:
Mentioned this with @amyb-hiveit, but we should be consistent and add the `toBeInTheDocument` in Jest tests